### ISSUE TITLE
Better way to manage lots of entries

### ISSUE THREAD

**Issue by nickbe** [zeruniverse/Password-Manager#70]
Therefore I suggest to add a way to group items by the first tag defined (which would be easy and convenient) 

btw. the tag cloud seems to have vanished completely. why is that?

**Comment by BenjaminHae** [comment]
It's because of the new implementation of the table.
Getting it back should be easy, I think I'll have time for it next week.

Grouping should be possible following this [example](https://datatables.net/examples/advanced_init/row_grouping.html) from datatables.

**Comment by nickbe** [comment]
Great. This helps a lot

**Comment by zeruniverse** [comment]
Did dfa698d2cedf0bf5f142e22780544785b6aafdaf solve this problem?

**Comment by nickbe** [comment]
Yes, tag cloud seems to be back now and working.
Will there be any problem upgrading from 9.08 to the current version?

**Comment by zeruniverse** [comment]
I did change the config file. But it seems the database structure is not touched. I suggest to make a backup anyway.

**Comment by zeruniverse** [comment]
And I put `-rc` as it's a pre-release. Not suitable for production.

**Comment by nickbe** [comment]
I'll wait for grouping anyway as it'll make handling a large database easier.

**Comment by BenjaminHae** [comment]
I think only the btw was solved. The grouping isn't done, so I'll reopen.

**Comment by BenjaminHae** [comment]
@nickbe  you could try https://github.com/BenjaminHae/Password-Manager/tree/grouping but there are a lot of bugs especially when filtering by tags ..

**Comment by nickbe** [comment]
I just tried this version :) That's already very good this way.
There weren't any obvious bugs - at least not at first sight. 

The only thing I would improve is the group words font size. I'd try a larger font here.

Another idea would be to make everything below a group word collabsible. And maybe with lots of password entries visible (I set mine to 100) show them pre-collabsed. Well as I said this is just a thought.

Anyway thanks for this.
It really helps coping with large numbers of passwords.
